cfme/scripting/sprout.py

```diff
@@ -100,13 +100,14 @@ def populate_config_from_appliances(appliance_data):
 
     y_data['appliances'] = []
     for app in appliance_data:
-        y_data['appliances'].append({'base_url': 'https://{}/'.format(app['ip_address'])})
+        app_config = dict(
+            hostname=app['ip_address'],
```

Comment:
So I'm cool with deprecating plain base_url support - but I don't see the need for changing to 'hostname' and passing ui_protocol.

This has an effect on some downstream systems that will need updating to reflect an actual change from base_url to hostname. Things that are generating env files and such.

Reply:
PRT runs from the dockerbot file you updated so I think that's good, this might be a good opportunity to move some of that logic into the `miq` command set, for writing an env.yaml file based on given options.